src/components/HistoryTables/MessageHistory/useAge.ts

```diff
@@ -20,7 +20,7 @@ export const useUnformattedDateTime = (
 
   useChainHeadSubscription({
     variables: {},
-    skip: !message,
+    skip: !message || !!message?.block?.Timestamp,
```

Comment:
Just want to make sure this is intentional and doesn't introduce unexpected behavior - 

this says to me "dont subscribe to chain head if the message has a timestamp"

Do we know for sure that `!!message?.block?.Timestamp` evaluates to truthy/falsey when we expect it to? One way to test this is just submit a transaction, open it up immediately, and watch it go from pending -> confirmed state

then open a new table on the explorer, and refresh _immediately_ after the message confirms in original UI step above ^^. Both should show a correct looking time (they won't be exactly the same, but close enough is fine)

Reply:
Well it's a bit funny, when I view the transaction locally it's always a bit more slow than when I have the same transaction open on the wallet at https://wallet-calibration.glif.link/

But both of them actually first show ~1 year ago and then jump to the actual time. On the remote version, this jump goes a lot faster. Undoing my change above doesn't have any influence on this.

The reason for the change is that we do the exact same check in the return statement:
```
if (!!message?.block?.Timestamp) {
  return dayjs.unix(message.block?.Timestamp)
}
```

So if `!!message?.block?.Timestamp` is truthy, we don't use the result of the subscription anyway